common/scala/src/main/resources/reference.conf

```diff
@@ -82,3 +82,22 @@ dispatchers {
     throughput = 5
   }
 }
+
+whisk {
+  kafka-activation-store {
+    # Name of the Kafka topic for sending WhiskActivations
+    # Set to
+    # - `completed-others` - When LogDriverLogStore is used
+    # - `activations` - When other LogStore is used where logs are stored as part of activation result
+    activations-topic = "completed-others"
```

Comment:
I feel there may be a more intuitive name for this topic(`completed-others`).
It makes me vision activations sent to some misc controllers or else at first sight.

I am not quite sure it's worth to do and I know this is to use the topic pattern, but how about using a more intuitive name? such as `activations-with-logs` and just `activations`? 

Opinion from others would be appreciated.

Reply:
Was also thinking this about the name. Would `completed-async` work? It's at the very least much more descriptive than `others`